frameworks/helloworld/tests/test_sanity.py

```diff
@@ -32,12 +32,14 @@ def configure_package(configure_security):
     try:
         sdk_install.uninstall(FOLDERED_SERVICE_NAME, package_name=PACKAGE_NAME)
 
+        service_config = {"service": {"name": FOLDERED_SERVICE_NAME, "user": "root"}}
```

Comment:
Is this the way to do this here? If we don't specify "root" as the user, then the upgrade fails, but it feels as if this is not the ideal solution since we don't want to only test root installs forever.

Thoughts @benclarkwood @NimaVaziri?